pyroclast/luigi/resources/cpu.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Rename to `crane_cpu.sh`. Same with gpu. Then have another resource that is `local_cpu`. No need for preambles in the luigi directory.